app/src/main/res/values-night/themes.xml

```diff
@@ -2,12 +2,12 @@
     <!-- Base application theme. -->
     <style name="Theme.3048CarApp" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
         <!-- Primary brand color. -->
-        <item name="colorPrimary">@color/purple_200</item>
-        <item name="colorPrimaryVariant">@color/purple_700</item>
+        <item name="colorPrimary">@color/primary_light</item>
+        <item name="colorPrimaryVariant">@color/primary_dark_dark</item>
```

Comment:
Refactored color names to express their role in the color scheme, for clarity, and to follow these conventions.\
https://codepen.io/nicetransition/pen/QjwbRg